delta/plugins/statistics/src/main/resources/elasticsearch/mappings.json

```diff
@@ -0,0 +1,50 @@
+{
+  "properties": {
+    "@id": {
+      "type": "keyword"
+    },
+    "@type": {
+      "type": "keyword"
+    },
+    "properties": {
```

Comment:
If that's the case, that field should also be indexed, since the updateByQuery would have to do something like:
```json
{
  "query": {
    "nested": {
      "path": "relationshipCandidates",
      "query": {
        "term": {
          "relationshipCandidates.@id": "{resource_id}"
        }
      }
    }
  },
  "script": {
    "source": "for (prop in ctx._source. relationshipCandidates) {if(prop['@id'] == params.id) {if(ctx._source.relationships == null) ctx._source.relationships = new ArrayList(); prop['@type'] = params.types; } }",
    "lang": "painless",
    "params": {
      "id": "{{resource_id}}",
      "types": [ {{{resource_types}}} ]
    }
  }
}
```

Reply:
Yes, it would only bring a real boost for large resources